src/components/Global/Input/index.tsx

```diff
@@ -10,13 +10,46 @@ const Input = ({
   maxLength,
   minLength,
   labelText,
-  setInputVal
+  value,
+  setInputVal,
+  disabled,
+  onClick
 }: InputProps) => {
   const [input, setInput] = useState('');
 
   useEffect(() => {
     setInputVal && setInputVal(input);
   }, [input, setInputVal]);
+
+  if (value && setInputVal)
+    return (
+      <>
+        <div className={styles.container}>
+          {labelText && (
+            <label htmlFor={id} className={styles.label}>
+              {labelText}
+            </label>
+          )}
+          <input
+            className={styles.input}
+            type={type}
+            id={id}
+            name={id}
+            required={required}
+            placeholder={placeholder}
+            maxLength={maxLength}
+            minLength={minLength}
+            value={value}
+            onChange={event => {
+              setInputVal(event.target.value);
+            }}
+            disabled={disabled} // This will be false if disabled is not passed in
```

Comment:
Where is the logic that makes it false by default?

Reply:
If `disabled` is not passed, it will be undefined, which is falsy, it can only be true if someone actually pass `disabled=true`

Reply:
Roger that